src/Common/MemoryTracker.cpp

```diff
@@ -600,6 +600,21 @@ void MemoryTracker::setParent(MemoryTracker * elem)
         DB::current_thread->flushUntrackedMemory();
 
     parent.store(elem, std::memory_order_relaxed);
+
+    // FIXME: only for debug builds - NDEBUG
+#if 1
```

Comment:
```suggestion
#ifdef DEBUG_OR_SANITIZER_BUILD
```

Reply:
Yes, but first, I wanted to make sure that it does not have any such places, hence under `#if 1`. Applied.